### PR TITLE
Fix references to closed tickets

### PR DESCRIPTION
**Pull request by Wumpf**
### What

the usual cleanup!

### Checklist
* [x] pass main ci - I re-enabled a snippet test!
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7219?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7219?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!
* [x] If have noted any breaking changes to the log API in `CHANGELOG.md` and the migration guide

- [PR Build Summary](https://build.rerun.io/pr/7219)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

**Review by emilk — APPROVED** [review]

**Comment by emilk** [comment]
@rerun-bot full-check

File: crates/store/re_types_core/src/tuid.rs [hunk 1]
```diff
@@ -64,8 +64,7 @@ impl Loggable for Tuid {
         ];
         let validity = None;
 
-        // TODO(#3360): We use the extended type here because we rely on it for formatting.
-        Ok(StructArray::new(Self::extended_arrow_datatype(), values, validity).boxed())
+        Ok(StructArray::new(Self::arrow_datatype(), values, validity).boxed())
```
Comment (Wumpf) [inline crates/store/re_types_core/src/tuid.rs:67]: @teh-cmc this change here causes the tests to fail against their snapshot with:

```diff
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────
-old snapshot
+new results
────────────┬────────────────────────────────────────────────────────────────────────────────────────────────────
    0     0 │ [
    1     1 │     "1048576 scalars",
    2     2 │     "41.5 MiB in total",
    3       │-    "42 B per row",
          3 │+    "41 B per row",
    4     4 │ ]
```

which sounds like a good thing ™️ and I also think the change still makes sense.
Can you confirm?

Comment (emilk) [reply crates/store/re_types_core/src/tuid.rs:67]: Are Tuid:s still formatted correctly?

Comment (Wumpf) [reply crates/store/re_types_core/src/tuid.rs:67]: image
this? looks ok

Comment (Wumpf) [reply crates/store/re_types_core/src/tuid.rs:67]: via clement in person: don't do this! change it it to a TODO(cmc). Display on rrd dump is affected

**Comment by Wumpf** [comment]
@rerun-bot full-check

**Pull request merged**